lib/matplotlib/tests/test_table.py

```diff
@@ -79,3 +79,25 @@ def test_label_colours():
               colColours=colours,
               colLabels=['Header'] * dim,
               loc='best')
+
+
+@image_comparison(baseline_images=['table_cell_manipulation'],
```

Comment:
Add the `remove_text=True` kwarg here to simplify the figures, and reduce the likelihood of spurious differences related to font handling.